pretyping/tacred.ml

```diff
@@ -984,7 +984,7 @@ let e_contextually byhead (occs,c) f = { e_redfun = begin fun env sigma t ->
     if nowhere_except_in && (!pos > maxocc) then (* Shortcut *) t
     else
     try
-      let subst =
+      let (_sigma,subst) = (* FIXME *)
```

Comment:
It seems that the interpretation of the pattern in Tacinterp did not add the universes to the sigma, so if adding the universe constraints to the sigma needs the universes to be declared, there may be problems.

Then, whether to update the sigma or not is a question of semantics to choose. If it is updated then `change (... ?A ... ?A ... ) with ( ... )` and `?A` matches `Type`, this will force the unification of the univers on the fly, won't it?

If it is not, then the `change`'s pattern will be considered as a matching pattern, not a unification pattern.